## Z scoring 

In [5]:
import pandas as pd 
import numpy as np 
import os 
import sys 
import csv 


In [6]:

url = "https://raw.githubusercontent.com/fedhere/PUI2018_fb55/master/Lab4_fb55/times.txt"

In [7]:
Bus = pd.read_csv(url, sep=",", names=['Commute Time'])

In [8]:
Bus.head()

Commute Time
0     31.622239
1     32.821376
2     30.229101
3     31.413766
4     39.010550

In [9]:
m = Bus['Commute Time'].mean()
std = Bus['Commute Time'].std()
N = len(Bus)
print ("Mean:", m, "\nStandard Deviation:", std, "\nN", N)

Mean: 34.4661616883 
Standard Deviation: 7.13728013375 
N 100


In [10]:
MeanP = 36
StDev = 6

In [11]:
Num = MeanP-m
Den = StDev/np.sqrt(N)
Z = Num/Den
print('Z-test result:', Z)

Z-test result: 2.55639718617


## Result interpretation:
Z test results reflect that 2.556 larger than 2; with a previous choice of 0.05 as the significance level which parallels to the 2 sigma. Therefore, the null hypothesis is rejected. 